core/src/test/scala/anorm/JavaTimeSpec.scala

```diff
@@ -188,6 +188,15 @@ class JavaTimeColumnSpec extends Specification {
           aka("parsed zoned date/time") must_=== date
       }
 
+    "be parsed from timestamp with nano precision" in {
+      val instantWithNanoPrecision = Instant.parse("2021-01-06T08:45:26.441477Z")
```

Comment:
Is it me or the tested instant has a millisecond precision instead of nanosecond?

Reply:
The tested instant has `441477` **micro**seconds. The issue was happening for all the presisions under millisecond. So the test is not wrong, just the naming.